CMake/folly-deps.cmake

```diff
@@ -118,6 +118,13 @@ if(ZSTD_FOUND)
   list(APPEND FOLLY_LINK_LIBRARIES ${ZSTD_LIBRARY})
 endif()
 
+find_package(Xxhash MODULE)
+set(FOLLY_HAVE_LIBXXHASH ${XXHASH_FOUND})
+if(XXHASH_FOUND)
```

Comment:
In a second thought, the build will fail if xxhash is not found. Therefore is preferable to fail/abort on the configure stage, before build.

Reply:
Done.